src/v/test_utils/test_macros.h

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Redpanda Data, Inc.
```

Comment:
I don't know if the year matters but thankfully we're past this year 😆

Reply:
Done